web/src/api/zfcp.ts

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
IMHO, `zftp` should be `storage/zfcp` (as the `iscsi` case). The same for `dasd` (although I do not expect `dasd` to be moved in this PR).

Reply:
Already did for the WEB components, so it is ok for moving also in this case.

Reply:
Already did for the components, so, it is fine doing it for both cases at the same time.